<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pyFillGaps</class>
 <widget class="QMainWindow" name="pyFillGaps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyFillGaps</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pyPmedia/fill.PNG</normaloff>:/pyPmedia/fill.PNG</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>811</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCSV"/>
   <addaction name="actionSHP"/>
   <addaction name="actionFalhas"/>
   <addaction name="actionSalvarCSV"/>
  </widget>
  <action name="actionCSV">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pyPmedia/csvIco.png</normaloff>:/pyPmedia/csvIco.png</iconset>
   </property>
   <property name="text">
    <string>Carregar CSV...</string>
   </property>
   <property name="toolTip">
    <string>Carregar arquivo CSV</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionSHP">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pyPmedia/shpIcon.png</normaloff>:/pyPmedia/shpIcon.png</iconset>
   </property>
   <property name="text">
    <string>Carregar SHP...</string>
   </property>
   <property name="toolTip">
    <string>Carregar shapefile das estações</string>
   </property>
  </action>
  <action name="actionFalhas">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pyPmedia/fill.PNG</normaloff>:/pyPmedia/fill.PNG</iconset>
   </property>
   <property name="text">
    <string>Preenchimento de falhas</string>
   </property>
  </action>
  <action name="actionSalvarCSV">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pyPmedia/salvar.png</normaloff>:/pyPmedia/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar CSV</string>
   </property>
   <property name="toolTip">
    <string>Salvar arquivo CSV</string>
   </property>
  </action>
  <action name="actionCreditos">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pyPmedia/logoDHB.png</normaloff>:/pyPmedia/logoDHB.png</iconset>
   </property>
   <property name="text">
    <string>Créditos</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
